terraform/environment/ecs_front.tf

```
@@ -233,7 +233,8 @@ locals {
         { "name" : "OPG_LPA_COMMON_PDF_CACHE_S3_BUCKET", "value" : data.aws_s3_bucket.lpa_pdf_cache.bucket },
         { "name" : "OPG_LPA_COMMON_PDF_QUEUE_URL", "value" : aws_sqs_queue.pdf_fifo_queue.id },
         { "name" : "OPG_LPA_ENDPOINTS_API", "value" : "http://${local.api_service_fqdn}" },
-        { "name" : "OPG_LPA_OS_PLACES_HUB_ENDPOINT", "value" : "https://api.os.uk/search/places/v1/postcode" }
+        { "name" : "OPG_LPA_OS_PLACES_HUB_ENDPOINT", "value" : "https://api.os.uk/search/places/v1/postcode" },
+        { "name" : "OPG_LPA_COMMON_REDIS_CACHE_URL", "value" : data.aws_elasticache_cluster.redis_cache.cache_nodes.0.address}
```

```suggestion
        { "name" : "OPG_LPA_COMMON_REDIS_CACHE_URL", "value" : "tls://${data.aws_elasticache_cluster.redis_cache.primary_endpoint_address}"
```
similar to this line: https://github.com/ministryofjustice/opg-use-an-lpa/blob/0ecc79f7de30c9e8ef3bc4ce5280cb48780fb02b/terraform/environment/actor_ecs.tf#L231